package.json

```diff
@@ -38,7 +38,6 @@
   },
   "dependencies": {
     "prismarine-block": "^1.0.0",
-    "protodef": "^1.3.1",
```

Comment:
well ok but shouldn't you add a dep to protodefc ?

Reply:
No, it's not published we need my ProtoDefc stuff merged and my node-protodefc stuff merged. Use npm link.